python/GafferUI/TweakPlugValueWidget.py

```diff
@@ -171,6 +172,8 @@ def __validModes( plug ) :
 	result = []
 	if Gaffer.Metadata.value( plug.parent(), "tweakPlugValueWidget:allowCreate" ) :
 		result += [ Gaffer.TweakPlug.Mode.Create ]
+	if Gaffer.Metadata.value( plug.parent(), "tweakPlugValueWidget:allowCreateDefault" ) :
```

Comment:
Do you have any use cases in mind where `Create` would be wanted but `CreateDefault` wouldn't? This `allowFoo` pattern is a bit clunky, and I'd prefer not to extend it further if possible. I suggest we just apply `allowCreate` to `CreateDefault`, and consider moving to a more general mechanism in future (for instance, a pair `includedModes` and `excludedModes` settings, each accepting wildcards).

Reply:
> Do you have any use cases in mind where Create would be wanted but CreateDefault wouldn't?

Having both maybe gets a little muddied with the `ignoreMissing` modes as we mentioned briefly back in https://github.com/GafferHQ/gaffer/pull/4733#discussion_r918185122, but now that we have a specific request for that, I think it's fine including both with `allowCreate`.

Fixed in https://github.com/GafferHQ/gaffer/pull/5047/commits/19418b4fa6215cc4afabe83fbd08f1c8acb24563